tests/tar/tar_writer_test.py

```diff
@@ -217,6 +217,7 @@ def testPackageDirAttribute(self):
     expected_content = [
         {"name": "my"},
         {"name": "my/package"},
+        {"name": "my/package/mylink"},
```

Comment:
It seems that this behavior in some cases might necessitate a tight binding between what's in `package_dir` and the contents of `pkg_mklink` if a symlink needs to reference something relative to the base of the package.

Thoughts?  This could perhaps be fixed via a template.

Reply:
I think the case you have is  
- ` mk_link(foo -> bar)`
- `packgae_dir = baz`

Gives: ` baz/foo -> bar`, but we want `baz/foo => baz/bar`.

I'm going to call that a distinct feature request.  I can see use cases for both leaving the link relative (or absolute) and for re-rooting it.